### PR TITLE
MEP12 on load_converter.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/load_converter.py [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
-from matplotlib.dates import bytespdate2num
-#from matplotlib.mlab import load
 import numpy as np
-from pylab import figure, show
+import matplotlib.pyplot as plt
 import matplotlib.cbook as cbook
+import matplotlib.dates as mdates
+
+
+# Note: matplotlib.dates doesn't have bytespdate2num.
```
Comment (jenshnielsen) [inline examples/pylab_examples/load_converter.py]: I'm confused about this. My version of Matplotlib definitely has bytespdate2num

Comment (ericmjl) [inline examples/pylab_examples/load_converter.py]: Ok, let me try conda installing the latest MPL.

Comment (jenshnielsen) [inline examples/pylab_examples/load_converter.py]: It was added in February https://github.com/matplotlib/matplotlib/commit/0a24a56ea536698ca25572d4e089b6830ef346f4 so it wound be in any released version.

Comment (ericmjl) [inline examples/pylab_examples/load_converter.py]: Very weird, I cannot make the import statements work, though. I get the error: 

    In [9]: import matplotlib.dates as mdate

    In [10]: mdate.bytespdate2num()
    ---------------------------------------------------------------------------
    AttributeError                            Traceback (most recent call last)
    <ipython-input-10-7c2709974f11> in <module>()
    ----> 1 mdate.bytespdate2num()

    AttributeError: 'module' object has no attribute 'bytespdate2num'

Any idea what's going on? Is the import statement not done correctly?

**Comment by jenshnielsen** [comment]
Which version of Matplotlib are you using? You will need the master branch with a build from after February for this to work.

**Comment by ericmjl** [comment]
I'm on 1.4.3 right now.

**Comment by ericmjl** [comment]
Should I be doing a development install from master instead?

**Comment by jenshnielsen** [comment]
Yes if you want to run this example locally you need to do that

**Comment by ericmjl** [comment]
Ok, on it.

**Comment by ericmjl** [comment]
Very interesting, @jenshnielsen, I am finding that with `1.4.3` installed, I still have issues with the import statements. Also, when I perform a local install from the repo directly:

    python setup.py develop

I get the following messages:

    IMPORTANT WARNING:
        pkg-config is not installed.
        matplotlib may not be able to find some of its dependencies
    ============================================================================
    Edit setup.cfg to change the build options

    BUILDING MATPLOTLIB
                matplotlib: yes [1.5.dev1]
                    python: yes [3.4.3 |Continuum Analytics, Inc.| (default, Mar
                            6 2015, 12:07:41)  [GCC 4.2.1 (Apple Inc. build
                            5577)]]
                  platform: yes [darwin]

    REQUIRED DEPENDENCIES AND EXTENSIONS
                     numpy: yes [version 1.9.2]
                  dateutil: yes [using dateutil version 2.4.2]
                      pytz: yes [using pytz version 2015.4]
                   tornado: yes [using tornado version 4.2.1]
                 pyparsing: yes [using pyparsing version 2.0.3]
                    libagg: yes [pkg-config information for 'libagg' could not
                            be found. Using local copy.]
                  freetype: no  [The C/C++ header for freetype2 (ft2build.h)
                            could not be found.  You may need to install the
                            development package.]
                       png: yes [version 1.6.17]
                     qhull: yes [pkg-config information for 'qhull' could not be
                            found. Using local copy.]

    OPTIONAL SUBPACKAGES
               sample_data: yes [installing]
                  toolkits: yes [installing]
                     tests: yes [nose 0.11.1 or later is required to run the
                            matplotlib test suite. "setup.py test" will
                            automatically download it. Install nose 0.11.1 or
                            later to run matplotlib.test() / using
                            unittest.mock]
            toolkits_tests: yes [nose 0.11.1 or later is required to run the
                            matplotlib test suite. "setup.py test" will
                            automatically download it. Install nose 0.11.1 or
                            later to run matplotlib.test() / using
                            unittest.mock]

    OPTIONAL BACKEND EXTENSIONS
                    macosx: yes [installing, darwin]
                    qt5agg: no  [PyQt5 not found]
                    qt4agg: yes [installing, Qt: 4.8.6, PyQt: 4.8.6; PySide not
                            found]
                   gtk3agg: no  [Requires pygobject to be installed.]
                 gtk3cairo: no  [Requires cairocffi or pycairo to be installed.]
                    gtkagg: no  [Requires pygtk]
                     tkagg: yes [installing, version not identified]
                     wxagg: no  [requires wxPython]
                       gtk: no  [Requires pygtk]
                       agg: yes [installing]
                     cairo: no  [cairocffi or pycairo not found]
                 windowing: no  [Microsoft Windows only]

    OPTIONAL LATEX DEPENDENCIES
                    dvipng: yes [version 1.14]
               ghostscript: yes [version 9.10]
                     latex: yes [version 3.14159265]
                   pdftops: no

    ============================================================================
                            * The following required packages can not be built:
                            * freetype

Cannot tell whether the installation was successful or not. Any ideas here? I am at a loss with this one.

**Comment by jenshnielsen** [comment]
It is expected to not work with 1.4.3 the code was not added to the master branch before February this year. 

To install Matplotlib from source you need to have pkg-config installed. On OSX i normally install it using homebrew but that really depends on how your machine is set up.

We really should have a Conda branch with a master build but unfortunately we don't have that at the moment.

**Pull request merged**